src/Content/Widget/VCard.php

```diff
@@ -99,6 +100,10 @@ public static function getHTML(array $contact): string
 			if (in_array($rel, [Contact::FOLLOWER, Contact::FRIEND]) && Contact::canReceivePrivateMessages($contact)) {
 				$wallmessage_link = 'message/new/' . $id;
 			}
+
+			if (in_array($rel, [Contact::SHARING])) {
```

Comment:
Are you specifically testing whether the contact is a group here? If yes, this condition seems too wide, it would match all the contacts you follow.

Reply:
yes that's right. I'm testing if a contact is a group... 
how can i test this faster and better in this place?

Reply:
```suggestion
			if ($contact['contact-type'] == Contact::TYPE_COMMUNITY) {
```

Reply:
thank you!
changed it.